### PR TITLE
Add new UI for verified challenges

### PR DESCRIPTION
**Pull request by piazzatron**
### Description

Adds a number of changes for referrals & referrals-verified challenges, documented on Figma and in Linear
- New share to twitter button
- For aggregate challenges, show amount as the max you'd get if every referral was completed
- Some verbiage changes for the verified version of the challenge

Screen Shot 2022-01-25 at 7 21 24 PM

### Dragons

- I should probably QA this myself again tomorrow with less covid-brain
- Still needs native mobile changes

### How Has This Been Tested?

- Tested with nonverified + verified acounts

### How will this change be monitored?

For features that are critical or could fail silently please describe the monitoring/alerting being added.

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/aud-1365

File: src/pages/audio-rewards-page/ChallengeRewardsTile.tsx [hunk 1]
```diff
@@ -37,10 +37,18 @@ const messages = {
   claimReward: 'Claim Your Reward'
 }
 
+// For aggregate challenges, we show the total amount
+// you'd get when completing every step of the challenge
+// -- i.e. for referrals, show 1 audio x 5 steps = 5 audio
+export const getAmount = (challenge?: OptimisticUserChallenge) =>
```
[the remark this answers is not in the document]
Comment (piazzatron) [reply src/pages/audio-rewards-page/ChallengeRewardsTile.tsx:43]: will move to hook 👍

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/aud-1365

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/aud-1365

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/aud-1365

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/aud-1365

**Review by raymondjacobson — APPROVED** [review]

**Pull request merged**